src/tike/view.py

```diff
@@ -80,56 +83,77 @@ def complexHSV_to_RGB( img0 ):
     Returns
     -------
     rgb_img : :py:class:`numpy.array`
-        The (..., 3) shaped array which represents the input complex valued array 
+        The (..., 3) shaped array which represents the input complex valued array
         in a RGB colorspace.
     """
-    
+
     sz = img0.shape
 
-    hsv_img = np.ones( ( *sz, 3 ), 'float32' )
+    hsv_img = np.ones((*sz, 3), 'float32')
+
+    hsv_img[ ..., 0 ] = np.angle( img0 )     
+    hsv_img[ ..., 2 ] = np.abs( img0 )
+
+    #================================
+    # Rescale hue to the range [0, 1]
+
+    eps = np.finfo( np.float32 ).eps
 
-    # Rescale hue and value to the range [0, 1].
-    hsv_img[ ..., 0 ] = np.angle( img0 ) 
     hsv_img[ ..., 0 ] -= np.min( hsv_img[ ..., 0 ] )
-    hsv_img[ ..., 0 ] = hsv_img[ ..., 0 ] / ( np.finfo(np.float32).eps + np.max( hsv_img[ ..., 0 ] ))
+    hsv_img[ ..., 0 ] = hsv_img[ ..., 0 ] / ( eps + np.max( hsv_img[ ..., 0 ] ))
+    
+    #=================================================================
+    # NOTE: WHAT HAPPENS IF WE GIVE AN IMAGE WITH ONLY PHASE CONTRAST?
+    # THIS WILL SET MAGNITUDE TO ZERO EVERYWHERE...NOT WHAT WE WANT!!!
+    #
+    # I WILL ASSUME THAT THE USER WILL RESCALE THEIR ARRAY TO [ 0, 1 ]
+    # PRIOR TO CALLING THIS FUNCTION
+    #=================================================================
```

Comment:
You cannot have a wavefront with magnitude zero everywhere. That would be no wave and there would be no phase information.

Reply:
np.abs always returns values in [0, inf), so you should just divide by max. Raise an error if max is 0.